pyproject.toml

```diff
@@ -16,6 +16,9 @@ llama-index = { extras = ["local_models"], version = "0.9.3" }
 watchdog = "^3.0.0"
 qdrant-client = "^1.6.9"
 chromadb = {version = "^0.4.13", optional = true}
+gradio = "4.4.1"
+pymysql = "^1.1.0"
+cryptography = "^41.0.7"
```

Comment:
This has not its place in the default dependencies. `gradio` is in the `ui` target.

Additionally, I think that `pymsql` should not be added in the dependencies of this project, as this is a driver that is specific to mySQL: users might not use mySQL, and might need other dependencies (`psycopg2`, `psycopg`, and many others for example - c.f. sqlalchemy dialects)